tests/integration/cmd_dev_test.go

```diff
@@ -2373,7 +2373,7 @@ CMD ["npm", "start"]
 			devfileCmpName := "nodejs"
 			BeforeEach(func() {
 				if podman {
-					Skip("Not implemented yet on Podman - see https://github.com/redhat-developer/odo/issues/6493")
+					Skip("Getting annotations is not available with Podman v3")
```

Comment:
Would it be possible to be more specific (as we are able to determine the actual version of Podman) and skip the test only if it is < v4? So that we can still have this tested with Podman v4..
I am perfectly okay if you think it is not worth the effort for now.

It would be great to look into how we could have tests on both Podman v3 and v4, but that's the scope of https://github.com/redhat-developer/odo/issues/6583